sdk/dnsresolver/Microsoft.Azure.Management.DnsResolver/tests/Microsoft.Azure.Management.DnsResolver.Tests.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.test.reference.props" />
+  <PropertyGroup>
+    <PackageId>DnsResolver.Tests</PackageId>
+    <Description>DnsResolver.Tests Class Library</Description>
+    <AssemblyName>DnsResolver.Tests</AssemblyName>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <Version>1.0.0</Version>    
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="FluentAssertions" Version="5.6.0" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="14.0.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.6.0-preview" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\src\Microsoft.Azure.Management.DnsResolver.csproj" />
```

Comment:
Use absolute references.

Reply:
It's the pattern in this repo. 
https://github.com/Azure/azure-sdk-for-net/search?p=9&q=ProjectReference